src/routes/home/components/AnnouncementDialog.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+  <BaseDialog
+    id="announcement-dialog"
+    v-model="show"
+    variant="light"
+    :opacity="1"
+    data-cy="announcement-dialog"
+    scrollable
+    :max-width="750"
+  >
+    <template #activator>
+      <v-btn
+        color="newSecondary"
+        width="full"
+        rounded="0"
+        block
+      >
+        Announcement Header
+      </v-btn>
+    </template>
+    <template #title>
+      <div color="primary" class="w-100">
+        <div class="d-flex justify-space-between w-100">
+          <h1 class="mb-4">
+            Generic Header
+          </h1>
+          <v-btn
+            variant="text"
+            icon
+            size="x-large"
+            @click="close"
+          >
+            <v-icon icon="mdi-close" />
+          </v-btn>
+        </div>
+      </div>
+    </template>
+    <template #body>
+      <div class="d-flex justify-center my-4">
+        <GameCard
+          v-for="card in displayCards"
+          :key="card.suit + card.rank"
+          :suit="card.suit"
+          :rank="card.rank"
+          class="card mb-10"
+          :high-elevation="true"
+        />
+      </div>
+      <p>Generic Announcement</p>
```

Comment:
When we pull this into a variable, same with title and the activator text, it might make sense to wrap them in a single object that has separate attributes for all these things e.g.

```
export const announcementData {
  id: 'valueToLookForInLocalStorage',
  activatorText: 'What shows up in the banner'.
  title: 'Heading in the dialog',
  imgSrc: 'coolPic.jpg',
  displayCards: [Card.ACE_OF_SPADES],
  announcementText: 'Body of the announcement',
}
```

Then this component can import those and assign them programmatically